### PR TITLE
creation of all models

### PR DESCRIPTION
**Pull request by pedrokogawa**
Creation of all models

### CODE REVIEW

File: app/models/textbook.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class Textbook < ApplicationRecord
   has_many :units
+  has_many :words, through: :units
```
Comment (super-shin) [inline app/models/textbook.rb:3]: do we need this? you made the words connection under units

File: app/models/user.rb [hunk 2]
```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
   has_one_attached :photo
+  has_many :classrooms_as_student, through: :attendances, source: :classrooms
+  has_many :classrooms
```
Comment (super-shin) [inline app/models/user.rb:4]: double associations on classroom?

File: app/models/user.rb [hunk 3]
```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
```
Comment (super-shin) [inline app/models/user.rb:1]: dont we need to add challenge and attendance associations?

**Review by super-shin — CHANGES_REQUESTED** [review]
@pedrokogawa  sorry just a bit of clarification on some of the code, thanks

File: app/models/user.rb (hunk 2 above)
Comment (pedrokogawa) [reply app/models/user.rb:4]: image
One would be the connection between user and classrooms as a teacher and the other is as student through attendance.

File: app/models/user.rb (hunk 3 above)
Comment (pedrokogawa) [reply app/models/user.rb:1]: Yes, you are right. I think I lost it when my computer turned off.

**Review by pedrokogawa — COMMENTED** [review]
I left some comments, could you check and correct the User model.

**Pull request merged**